extensions/extensions-docker/aissemble-configuration-store/pom.xml

```diff
@@ -39,10 +39,13 @@
                     </execution>
                 </executions>
             </plugin>
```

Comment:
I: we were configuring the fabric8 build incorrectly within the `extensions-docker` module (including an active run there).  These changes correct this issue so that we can use the updated configuration above.